### ISSUE TITLE
MinGW: Undeclared signals in energymon app

### ISSUE THREAD

**Issue by connorimes** [energymon/energymon#29]
This isn't the only compile problem for this app, but should have its own issue report given that other systems also may not recognize these signals.  Should check for these only on systems that recognize them.

<pre>
C:\development\git\energymon\src\app\energymon.c: In function 'shandle':
C:\development\git\energymon\src\app\energymon.c:33:10: error: 'SIGQUIT' undecla
red (first use in this function)
     case SIGQUIT:
          ^
C:\development\git\energymon\src\app\energymon.c:33:10: note: each undeclared id
entifier is reported only once for each function it appears in
C:\development\git\energymon\src\app\energymon.c:34:10: error: 'SIGKILL' undecla
red (first use in this function)
     case SIGKILL:
          ^
C:\development\git\energymon\src\app\energymon.c:35:10: error: 'SIGHUP' undeclar
ed (first use in this function)
     case SIGHUP:
          ^
</pre>

**Comment by connorimes** [comment]
Checks added in 236d098.
